eZ/Publish/API/Repository/Values/User/RoleCreateStruct.php

```diff
@@ -25,6 +25,20 @@
     public $identifier;
 
     /**
+     * True before the role has been published.
+     *
+     * @var bool
+     */
+    public $is_new = true;
```

Comment:
@andrerom Right. Though in the case of user/usergroup for example, the api has no draft structs. Neither does Content and ContentTypeGroup, but ContentType does. Of these, only Content and ContentType support editing. Not clear to me yet why there are no ContentDrafts. Excavation continues.